docs/more/debugging.md

```diff
@@ -6,7 +6,8 @@ title: Debugging
 
 We **strongly recommend** the [`console_error_panic`](https://github.com/rustwasm/console_error_panic_hook) 
 which catches `panic!`s and outputs them to the console. Unfortunately this is not compatible with 
-apps built using `cargo-web`.
+apps built using `cargo-web`. If you call `yew::initialize()` before mounting your root component,
+this will automatically be done for you.
```

Comment:
Not sure why you didn't use `yew::start_app()` here... What you wrote is technically correct but might end up confusing new users into doing this:
```rust
yew::initialize();
yew::start_app::<MyApp>();
```

What if we rewrote the entire section to read something like this:
```markdown

If you use `yew::start_app()` to run your root component,
it automatically sets up [`console_error_panic`](https://github.com/rustwasm/console_error_panic_hook)
which catches `panic!`s and outputs them to the console.

If you need more control over the mounting process you can call `yew::initialize()`
(this is the same function that is called by `yew::start_app` internally) or set it up manually.

Note: `console_error_panic` is incompatible with `stdweb`.
``` 

I hope this ensures that the reader is aware they most likely don't even need to change anything in their setup to use `console_error_panic`.

Reply:
Sounds good.